pages/_app.tsx

```diff
@@ -1,7 +1,8 @@
 import type { AppProps } from 'next/app';
 import type { ReactElement } from 'react';
 import wrapper from 'store';
-import 'styles/globals.css';
+import '../styles/globals.css';
+import '../styles/typography.css';
```

Comment:
Why have you imported the typography here despite you imported it in the globals file already?